Translations/en_US/flashes.ini

```
@@ -3,4 +3,5 @@ custom.item.error.items.not.found="Items %ids% were not found"
 custom.item.error.items.denied="The action has been denied for items %ids%"
 custom.item.unlinked="%itemName% (%itemId%) was unlinked from %entityType% %entityId%"
 custom.item.linked="%itemName% (%itemId%) was linked to %entityType% %entityId%"
-custom.object.error.used.in.segments="Custom object "%name%" (#%id%) cannot be deleted because it's used in the following Segment(s): %segments%."
\ No newline at end of file
+custom.object.error.used.in.segments="Custom object "%name%" (#%id%) cannot be deleted because it's used in the following Segment(s): %segments%."
+custom.item.notice.merged="The item with ID <a href='%url%' data-toggle='ajax' data-menu-link='%menu_link%'><strong>%uniqueHash%</strong></a> already exists and the one you just inserted was merged with the existing one."
```

It may be confusing to show the unique hash to the users as we do not show it anywhere else. I'd suggest something like:
```suggestion
custom.item.notice.merged="The item the same unique identifier values <a href='%url%' data-toggle='ajax'>already exists</a> and the one you just inserted was merged with the existing one."
```
Also, I don't see `%menu_link%` variable getting set when this string is being translated.